nise/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.0.18"
```

Comment:
lets make this 2.0.19

Reply:
@maskarb 
So, I was going off of the versions of koku-nise in pypi: https://pypi.org/project/koku-nise/#history

I think when we merged the 2.0 branch back to master, things got a little wonky with what is in pypi and what the master version is. I think jumping back up to `2.1.6` will bring us back into sync.

Reply:
In that case, let's jump 2.1, and go to 2.2.0